src/calculateRentalCost.js

```diff
@@ -4,7 +4,27 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
-}
+  
+    const PRICE_PER_DAY = 40;
+    const LONG_TERM = 7;
+    const LONG_TERM_DISCOUNT = 50;
+    const SHORT_TERM = 3;
+    const SHORT_TERM_DISCOUNT = 20;
+    
+    const basicPrice = PRICE_PER_DAY * days;
+    if (days >= LONG_TERM) {
+    return basicPrice - LONG_TERM_DISCOUNT;
+    }
+    
+    if (days >= SHORT_TERM) {
+    return basicPrice - SHORT_TERM_DISCOUNT;
+    }
+    
+    return basicPrice;
```

Comment:
There is an indentation issue with the `if` statements and their corresponding return statements (lines 15-16 and 19-20). Each level of code block should be indented to improve readability.